dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Voltage/SecondaryVoltageControl/Simplified/SecondaryVoltageControl.mo

```diff
@@ -20,76 +20,100 @@ model SecondaryVoltageControl "Model for simplified secondary voltage control"
   parameter Types.PerUnit Alpha "PI integral gain";
   parameter Types.PerUnit Beta "PI proportional gain";
   parameter Types.PerUnit DerLevelMaxPu "Level slope limitation in pu/min (base QNomAlt)";
+  parameter Boolean FreezingActivated = false "Whether the freezing functionnality is activated or not";
+  parameter Integer NbMaxGen = 50 "Maximum number of generators that can participate in the secondary voltage control";
 
   //Input variables
+  Modelica.Blocks.Interfaces.BooleanInput[NbMaxGen] limUQDown(start = limUQDown0) "Whether the minimum reactive power limits are reached or not (for each generator participating in the secondary voltage control)";
```

Comment:
It would be clearer to write :

"If true, the reactive power lower limits are reached (for each generator participating in the secondary voltage control)"